docs/cli/index.md

```diff
@@ -118,44 +118,41 @@ Ouput:
 
 ### Import labels to your project
 
-To import labels, provide a list of files or folders (you can also procide a csv file external_id and label's paths).
+To import labels, provide a list of files or folders.
 
-Label's files are json with the json_response to upload
-
-!!! Examples "CSV file template"
-    ```
-    external_id;json_response_path
-    asset1; ./labels/label_asset1.json
-    asset2;./labels/label_asset2.json
-    ```
+You can provide several paths separated by spaces.<br>
+Label files are JSON files containing labels in the Kili format: the value corresponding to the
+jsonResponse field of a label
+(see [here](https://docs.kili-technology.com/reference/export-classification) for example). <br>
+File's name must be equal to asset's external_id.
 
 ```
 kili project label \
-    --project-id $project_id \
-    --from-csv labels.csv
+    labels \
+    --project-id $project_id
```

Comment:
I find this part quite confusing. `label` and `labels` are too similar. We can replace `labels` by some filenames directly

Reply:
it was to show that we can directly give a folder. in the demo folder, the folder name is labels. I will rename it in the

Reply:
I renamed it ground_truths

Reply:
folder with new name reuploaded to GCS